setup.py

```diff
@@ -153,26 +186,25 @@ class install(install_):
         ('h5plugin-dir=', None,
          'Where to install filter plugins. Default %s.' % H5PLUGINS_DEFAULT),
     ]
+
     def initialize_options(self):
         install_.initialize_options(self)
         self.h5plugin = False
         self.h5plugin_dir = H5PLUGINS_DEFAULT
+
     def finalize_options(self):
         install_.finalize_options(self)
         if self.h5plugin not in ('0', '1', True, False):
             raise ValueError("Invalid h5plugin argument. Mut be '0' or '1'.")
         self.h5plugin = int(self.h5plugin)
         self.h5plugin_dir = path.abspath(self.h5plugin_dir)
+
     def run(self):
         install_.run(self)
         if self.h5plugin:
-            if H51811P:
-                pass
-            else:
+            if HDF5_PLUGIN_SUPPORT:
```

Comment:
This looks great all around. Way more robust way to setting things up.

I think because the issue above, the plugins aren't being built (this isn't tested by travis unfortunately, since the environment only has 1.8.11).